src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Automating New Relic Service Catalog onboarding with GitHub Integration to enhance user adoption
+tags:
+  - New Relic integrations
+  - Github integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Establishing a New Relic Service Catalog offers immense value but currently demands significant time and resource investments from users. The process involves detailed structuring of teams, user creation, assignments, and mapping of entity ownership, presenting a substantial barrier to entry. To mitigate this, we propose an automation solution that leverages data from external systems, such as GitHub, where team structures and associations are already managed. This integration aims to streamline the setup process, significantly reduce upfront efforts, and accelerate the adoption of New Relic's Service Catalog.
```

Comment:
Thanks for providing this context. This is great. I have made minor tweaks.

Implementing a New Relic Service Catalog can unlock considerable benefits, but at present, it requires a considerable investment of time and resources. You have to take care of team orchestration, user provisioning, role assignments, and entity ownership delineation, which create a significant threshold for entry. To alleviate this challenge, we suggest the adoption of an automation solution that capitalizes on existing team configurations and relationships maintained in external systems such as GitHub. This simplifies the initial configuration, dramatically lessen the required effort, and accelerates the adoption of the New Relic Service Catalog, making its powerful insights and organization tools more accessible to users.